spring-cloud-build/pom.xml

```diff
@@ -125,6 +128,13 @@ limitations under the License.
 					<artifactId>asciidoctor-maven-plugin</artifactId>
 					<version>1.5.2</version>
 				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<configuration>
+						<redirectTestOutputToFile>true</redirectTestOutputToFile>
```

Comment:
NP, I think the only one that would suffer is spring-cloud-zookeeper.  I'll take care of it after you make your change.